python/lwt_interface/example_c_module.c

```diff
@@ -38,7 +38,99 @@
 
 #include "tskit_lwt_interface.h"
 
+static PyObject * example_receiving(PyObject *self, PyObject *args) {
+    int err = -1;
+    PyObject* ret = NULL;
+    LightweightTableCollection *tables = NULL;
+    tsk_treeseq_t tree_seq;
+    tsk_tree_t tree;
+
+    memset(&tree, 0, sizeof(tsk_tree_t));
+    memset(&tree_seq, 0, sizeof(tsk_treeseq_t));
+    
+    /* Get the tables from the args */
+    if (!PyArg_ParseTuple(args, "O!", &LightweightTableCollectionType, &tables)) {
+        goto out;
+    }
+
+    /* Check that the tables are init'd to prevent seg faults */
+    if (LightweightTableCollection_check_state(tables) != 0) {
+        goto out;
+    }
+
+    /* Build a tree sequence from the tables */
+    err = tsk_treeseq_init(&tree_seq, tables->tables, 0);
+    if (err < 0) {
+        handle_tskit_error(err);
+        goto out;
+    }
+    
+    /* Get the first tree */
+    err = tsk_tree_init(&tree, &tree_seq, 0);
+    if (err < 0) {
+        handle_tskit_error(err);
+        goto out;
+    }
+    err = tsk_tree_first(&tree);
+    if (err < 0) {
+        handle_tskit_error(err);
+        goto out;
+    }
+    
+    /* Return true if the tree has more than one root */
+    if (tsk_tree_get_num_roots(&tree) > 1) {
+        ret = Py_True;
```

Comment:
I think you need to Py_INCREF(Py_True) here. Safer to do something like
``ret = Py_BuildValue("i", tsk_tree_get_num_roots(&tree) > 1)``

(Is there a constructor code for bool?)

Reply:
I went with `Py_BuildValue("O", condition ? Py_True: Py_False);` as there is no code for booleans. I've also fixed the bad `Py_None` just below. 
For some reason I had it in my head that these didn't need refcounting - but it is the exact opposite.